### PR TITLE
Issue 72 - Support React 18

### LINKED ISSUE #72: React Embed generates duplicated iframes
### How to reproduce:
1. Create NextJS app with `npx create-next-app`
2. Install React Embed pkg with `npm i @formsort/react-embed`
3. Load the EmbedFlow as per documentation https://github.com/formsort/oss/tree/master/packages/react-embed
4. The wrapper creates duplicated iframe nodes
Screenshot 2022-06-20 at 14 36 29

### Cause
This is due to React 18 in Strict mode with its double render.

### Workaround
- Downgrade to React 17
- Remove Strict mode on React 18
- Using the @formsort/web-embed-api and using ref to make sure it only loads once

### Implementation
Implement on the react wrapper (react-embed pkg) the ref object to make sure it only loads once.

for example:
Screenshot 2022-06-20 at 15 13 52

Or add a proper cleanup function:
https://beta.reactjs.org/learn/synchronizing-with-effects#how-to-handle-the-effect-firing-twice-in-development

### PR DESCRIPTION
**Pull request by atikenny**
Fixes #72 

In React Strict mode, the embed renders twice. This PR includes an `unloadFlow` on the Web Embed interface and uses it to remove the effects of double rendering.

### CODE REVIEW

File: packages/react-embed/examples/event-log/src/index.tsx [hunk 1]
```diff
@@ -17,6 +16,9 @@ interface LoggedEvent {
 
 const App = () => {
   const [loggedEvents, setLoggedEvents] = useState<LoggedEvent[]>([]);
+  useEffect(() => {
```
Comment (atikenny) [inline packages/react-embed/examples/event-log/src/index.tsx:19]: Test useEffect double rendering.

**Review by lizraeli — APPROVED** [review]

File: packages/react-embed/examples/event-log/src/index.tsx (hunk 1 above)
Comment (BYK) [reply packages/react-embed/examples/event-log/src/index.tsx:19]: Maybe add inline comments for this? It doesn't tell me much when I read the code alone.

File: packages/react-embed/package.json [hunk 2]
```diff
@@ -50,16 +51,16 @@
...
     "ts-jest": "^27.0.5",
     "typescript": "^4.6.3"
   },
   "dependencies": {
-    "@formsort/web-embed-api": "^2.2.3"
+    "@formsort/web-embed-api": "^2.3.0"
   },
   "peerDependencies": {
-    "react": "^16.13.0 || ^17.0.0"
+    "react": "^16.13.0 || ^17.0.0 || ^18.0.0"
```
Comment (BYK) [inline packages/react-embed/package.json:63]: Why not just 
```suggestion
    "react": ">=16.13.0, <19.0.0"
```

**Review by BYK — APPROVED** [review]
Love it!

**Pull request merged**